src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const link = document.querySelectorAll('.list-item__link');
```

Comment:
The variable 'link' is declared here but is not used before being redeclared inside the 'DOMContentLoaded' event listener. Consider removing this declaration if it's not needed outside the event listener.